### ISSUE TITLE
Remove or increase "Alt text" character limit

### ISSUE THREAD

**Issue by koalie** [w3c/w3c-website#576]
**Describe the issue**
The CMS limits (125 characters, while the limit to "fig caption" is 500) the amount of text that can be added as "alt text" to informative images. I've now hit that limit a few times via the "image media" flexible component. As a result, I'm unable to fully describe some images, in particular those that contain text. The alt text needs to include all text that is in an image, so that people who are blind get the same information. 

Can the "alt text" character limit be removed or increased? If the latter, can this be increased to at least 500?

Thank you!

**Example**
https://www.w3.org/blog/2023/happy-holidays-from-the-world-wide-web-consortium/
Because of the size limit, the current alt is "_W3C logo turned into W30C (1994-2024), Tim Berners-Lee quote &quot;the web is humanity connected by technology&quot; and our greeting._"

But if there were no limitation, the best alt text would be "_W3C logo turned into W30C. 1994-2024. Quote by Sir Tim Berners-Lee, inventor of the Web, which reads 'The Web is humanity connected by technology'. We wish you happy holidays and all the best for 2024! We're looking forward to what we'll accomplish together in 2024._"

**Comment by NicolaSaunders** [comment]
[Issue #302](https://github.com/w3c/w3c-website/issues/302) appears to be related to this.

**Comment by JamieC24** [comment]
This issue has been resolved locally, waiting on other work to be approved before deploying to dev for review

**Comment by vivienlacourba** [comment]
The Alt Text for an image now allows up to 1250 chars.
The Figure Caption allows for up to 500 chars.

![Image](https://github.com/user-attachments/assets/d4eb9578-796d-4311-982b-11023e9dfa59)
